images/kindnetd/cmd/kindnetd/masq.go

```diff
@@ -17,49 +17,42 @@ limitations under the License.
 package main
 
 import (
-	"bytes"
 	"fmt"
-	"strings"
 	"time"
 
-	utildbus "k8s.io/kubernetes/pkg/util/dbus"
-	utiliptables "k8s.io/kubernetes/pkg/util/iptables"
-	utilexec "k8s.io/utils/exec"
+	"github.com/coreos/go-iptables/iptables"
 )
 
 // NewIPMasqAgent returns a new IPMasqAgent
 func NewIPMasqAgent(ipv6 bool, noMasqueradeCIDRs []string) *IPMasqAgent {
-	execer := utilexec.New()
-	dbus := utildbus.New()
-	protocol := utiliptables.ProtocolIpv4
+	protocol := iptables.ProtocolIPv4
 	if ipv6 {
-		protocol = utiliptables.ProtocolIpv6
+		protocol = iptables.ProtocolIPv6
 	}
-	masqChain := utiliptables.Chain(masqChainName)
-	iptables := utiliptables.New(execer, dbus, protocol)
+	ipt, _ := iptables.NewWithProtocol(protocol)
 
 	// TODO: validate cidrs
 	return &IPMasqAgent{
-		iptables:          iptables,
-		masqChain:         masqChain,
+		iptables:          ipt,
+		masqChain:         masqChainName,
 		noMasqueradeCIDRs: noMasqueradeCIDRs,
 	}
 }
 
 // IPMasqAgent is based on https://github.com/kubernetes-incubator/ip-masq-agent
 // but collapsed into kindnetd and made ipv6 aware in an opionated and simplified
-// fashion
+// fashion using "github.com/coreos/go-iptables" instead of k8s/pkg/util/iptables
```

Comment:
Looks actively enough maintained

Reply:
is being used by the cni plugins, we have an indirect dependency right now anyway